ksqldb-rest-app/src/main/java/io/confluent/ksql/api/auth/BasicCallbackHandler.java

```diff
@@ -45,6 +46,9 @@ public void handle(final Callback[] callbacks)
       if (callback instanceof NameCallback) {
         final NameCallback nc = (NameCallback) callback;
         nc.setName(username);
+      } else if (callback instanceof ObjectCallback) {
```

Comment:
Can you update the unit tests for this? Ideally we'd also add an integration test with an actual LoginModule of interest but that might be more challenging. 

For historical context, we actually used to support this but support was removed in order to limit dependencies on Jetty: https://github.com/confluentinc/ksql/pull/5134/files#r413547894